### PR TITLE
explore-deploy-uri-actions

### PR DESCRIPTION
**Pull request by jimscarver**
lookup explore and create deploy uri actions

this is basic functionality the interface should have.

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -209,6 +209,10 @@ export const actions = {
     fields: { URI: { value: '', type: 'uri' } },
     filename: 'actions/lookupURI.rho',
   },
+  createURI: {
+    fields: { value: { value: '', type: 'number' } },
```
Comment (Bill-Keuntje) [inline src/actions.js:213]: The number of fields does not match the createURI.rho code.
Either ```type``` doesn't belong here or there should be a placeholder in createURI.rho

File: src/actions/createURI.rho [hunk 2]
```diff
@@ -0,0 +1,20 @@
+match ["value"] {
```
Comment (Bill-Keuntje) [inline src/actions/createURI.rho:1]: See https://github.com/rchain-community/rgov/pull/213/commits/123476642b0dcba3ab818f7d281ecafaa5e7b1da#r660080001

File: src/actions/createURI.rho [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+[value] => {
+new
+   return(`rho:rchain:deployId`),
+   insert(`rho:registry:insertArbitrary`),   
+   stdout(`rho:io:stdout`),
+   uriCh
+in {
+   insert!(
+
```
Comment (Bill-Keuntje) [inline src/actions/createURI.rho:10]: Not sure what this extra whitespace is for.

File: src/actions/lookupURI.rho [hunk 4]
```diff
@@ -1,8 +1,8 @@
 match ["?"] {
 [URI] => {
-new
-   lookup(`rho:registry:lookup`),
-   return(`rho:rchain:deployId`),
+new     /* use EXPLORE */
```
Comment (Bill-Keuntje) [inline src/actions/lookupURI.rho:3]: Please be aware that this comment will not highlighted correctly

File: src/actions/createURI.rho (hunk 3 above)
Comment (jimscarver) [reply src/actions/createURI.rho:10]: this is where value might be replaced with a large data structure, like a ballot or set of addresses.

File: src/actions/lookupURI.rho (hunk 4 above)
Comment (jimscarver) [reply src/actions/lookupURI.rho:3]: should it be // use explore?

Comment (Bill-Keuntje) [reply src/actions/lookupURI.rho:3]: Highlighting ```/* comments */``` with REGEX is stupidly complex.
So, yeah, go ahead and make it ```// always use EXPLORE``` or whatever text you like.

File: src/actions/lookupURI.rho [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
+   return,
```
Comment (Bill-Keuntje) [inline src/actions/lookupURI.rho:4]: I'm trying to remember correctly here. The reason to put return ahead of lookup is explore outputs things to the first declared name?

File: src/actions/createURI.rho (hunk 3 above)
Comment (Bill-Keuntje) [reply src/actions/createURI.rho:10]: Go ahead and add some kind of informative comment, please.

File: src/actions/lookupURI.rho (hunk 5 above)
Comment (jimscarver) [reply src/actions/lookupURI.rho:4]: correct.

**Review by Bill-Keuntje — APPROVED** [review]
Nice work.

**Pull request merged**